app/src/main/java/a/sboev/ru/playlistmaker/audioplayer/di/AudioPlayerModule.kt

```diff
@@ -0,0 +1,28 @@
+package a.sboev.ru.playlistmaker.audioplayer.di
+
+import a.sboev.ru.playlistmaker.audioplayer.data.MyMediaPlayer
+import a.sboev.ru.playlistmaker.audioplayer.data.Player
+import a.sboev.ru.playlistmaker.audioplayer.data.PlayerRepositoryImpl
+import a.sboev.ru.playlistmaker.audioplayer.domain.api.PlayerInterActor
+import a.sboev.ru.playlistmaker.audioplayer.domain.api.PlayerRepository
+import a.sboev.ru.playlistmaker.audioplayer.domain.impl.PlayerInterActorImpl
+import org.koin.core.parameter.parametersOf
+import org.koin.dsl.module
+
+val audioPlayerModule = module {
+
+    val params = parametersOf()
+
+    factory<Player> {
+        MyMediaPlayer(params.get())
+    }
+
+    factory<PlayerRepository> {
+        PlayerRepositoryImpl(get())
+    }
+
+    factory<PlayerInterActor> { p ->
+        params.insert(0, p.get())
+        PlayerInterActorImpl(get())
+    }
```

Comment:
Здесь можно обойтись без поля `params`, удалить его и оставить так:
```suggestion
    factory<PlayerInterActor> { p ->
        PlayerInterActorImpl(get())
    }
```
Но вообще здесь нигде параметры не нужны, т.к. они не используются. Тогда можно удалить и поле `params` и параметр лямбды `p`